lang/src/org/partiql/lang/eval/time/Time.kt

```diff
@@ -136,7 +136,7 @@ data class Time(val localTime: LocalTime, val precision: Int = MAX_PRECISION_FOR
      * Returns the seconds along with the fractional part of the second's value.
      */
     val secondsWithFractionalPart
-        get()  = BigDecimal(localTime.second + localTime.nano / NANOS_PER_SECOND)
+        get()  = localTime.second.toBigDecimal() + localTime.nano.toBigDecimal() / NANOS_PER_SECOND.toBigDecimal()
```

Comment:
Just to verify, none of these fields in `localTime` are `double`, right?  In other words, are they all `long`?

Reply:
They are all `int`.